### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by dinosaw** [dany-on-demand/mn#3]
I'm back (I'm sorrrry).

Basically I've been playing with this today and can't get it working and I'm not sure if I'm actually using it correctly.

So I do have it installed and running, and can navigate to 127.0.01:3016 and am greeted with the interface where I can pick a username and chat and all that works. However the video player, once hitting play, simply doesn't do anything unfortunately.

I verified that I have an H.264/AAC .mp4 file in /media.

This is what I get when I start up the application:

```
0|server   | 
0|server   | route:  b53bc280-a96a-4aad-a9d8-XXXXXX 
0|server   | password: 27a0a1a7-7815-4dfb-8ae2-XXXXXX
0|server   | 
0|server   | login route: b8d2ab5f-d2ff-4363-8909-XXXXXX
0|server   | 
0|server   | Warning: media file specified does not exist, add it to /media 
0|server   | listening on port *:3016 
0|server   | version 5dc411a
0|server   | 

[STREAMING] Now streaming realtime logs for [0] process
```

One thing I'm curious about is the route / password / login route. The README says to write them down, but I'm not entirely sure what to do with them. I thought they were some sort of login credentials, but when I navigate to 127.0.0.1:3016 I'm not prompted to log in (this is fine, I can restrict access in my firewall, but just curious what the routes and password are used for).

Also the warning indicates that it sees no media file in /media. Here is the probe info:

```
[root@dev media]# pwd
/media
[root@dev media]# ls
output.mp4
```

```
[root@dev media]# ffprobe output.mp4 
FFprobe version 0.6.5, Copyright (c) 2007-2010 the FFmpeg developers
  built on Jan 29 2012 17:52:15 with gcc 4.4.5 20110214 (Red Hat 4.4.5-6)
...
...
    Stream #0.0(eng): Video: h264, yuv420p, 1280x638 [PAR 1:1 DAR 640:319], 5926 kb/s, 23.98 fps, 23.98 tbr, 1k tbn, 47.95 tbc
    Stream #0.1(eng): Audio: aac, 48000 Hz, 5.1, s16, 199 kb/s
    Stream #0.2(eng): Subtitle: text / 0x74786574
```

Any ideas? If not no worries, you've already been a big enough help!

**Comment by dany-on-demand** [comment]
navigate to `127.0.0.1:3016/b8d2ab5f-d2ff-4363-8909-XXXXXX` and enter the password from the command line

this will redirect you to the admin route where you can change the settings from `/config/globals.json` on the fly

changing the media filename will update all connected clients so you can begin watching another movie in the /media folder, without making everyone refresh their browsers, it was seamless when I did it

it's an _unfriendly_ interface I admit, it was good enough for me and friends though

you can also just change the filename in `/config/globals.json`

the routes and password are regenerated each startup

this is a security through obscurity measure I did as a placeholder until I add in actual authentication

you need one person to be logged in as admin to press play on the video

the rest of the clients will sync up with the admin

**Comment by dinosaw** [comment]
Cool! Thank you for explaining the routing / login information.

So reporting back, everything is working great! I tested it with me and my friend and we had zero issues with time syncing on the media file. I will also embarrassingly admit, I thought `/media` was referring to the standard `/media` directory that Unix OSs come with lol so I didn't even have my file in the correct location to begin with. Then I noticed the `media` directory in the project folder and was like, wow... I'm so dumb haha. The admin "on-the-fly" filename changed works fine too.

But again, this is exactly what I wanted and it seems to be working great! I will test with a longer media file and 2-3 other friends at the same time. Also the UI isn't a big deal - I might darken it up a bit but other than that it's fine with me. Thanks again bud :)
